infra/load-balancer.tf

```diff
@@ -0,0 +1,25 @@
+resource "helm_release" "load_balancer_controller" {
+  name       = "jovand-loadbalancer-controller"
+  repository = "https://aws.github.io/eks-charts"
+  chart      = "aws-load-balancer-controller"
+  namespace  = "default"
+  set {
+    name  = "clusterName"
+    value = module.eks.cluster_name
+  }
+
```

Comment:
Add **region** value and please set **defaultTargetType** to "ip", more on what it does here: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.7/guide/ingress/annotations/#traffic-routing